python/pyspark/pandas/__init__.py

```diff
@@ -146,6 +146,9 @@ def _auto_patch_spark() -> None:
 
         df.DataFrame.to_pandas_on_spark = DataFrame.to_pandas_on_spark  # type: ignore
```

Comment:
@itholic can you create a JIRA and move this two methods to PySpark's Dataframe? we don't need to auto patch anymore

Reply:
Just created SPARK-35605 !